NVActivityIndicatorView/NVActivityIndicatorPresenter.swift

```diff
@@ -231,7 +239,15 @@ public final class NVActivityIndicatorPresenter {
             let bottomConstraint = NSLayoutConstraint(item: keyWindow, attribute: .bottom, relatedBy: .equal, toItem: containerView, attribute: .bottom, multiplier: 1, constant: 0)
 
             keyWindow.addConstraints([leadingConstraint, trailingConstraint, topConstraint, bottomConstraint])
-        }())
+            }())
+
+        if hasEnterAnimation {
+            containerView.alpha = 0
+
+            UIView.animate(withDuration: 1, delay: 0, options: [.curveEaseIn], animations: {
```

Comment:
It seems `enterExitAnimationDuration` should be used here instead of hardcoded `1`.